resources/publish.rb

```diff
@@ -24,13 +24,20 @@
 property :prefix,        String, default: ''
 property :distribution,  String, default: ''
 property :timeout,       Integer, default: 3600
+# Uses the given GPG passphrase. If you let aptly handle the whole GPG key-pair
+# creation mechanism, this is just whatever you the passphrase node attribute
+# is. In the case that you have replaced the key-pair with one that you
+# generated yourself and you don't have the password at the chef recipe compile
+# time (so you can't simply override the node attribute), you can pass the
+# passphrase explicitly here.
```

Comment:
I removed the comment. I think it would be nice to have something there as the only scenario where one would need this parameter is when doing something non-standard.